lib/pg_party/cache.rb

```diff
@@ -0,0 +1,53 @@
+require "thread"
+
+module PgParty
+  class Cache
+    LOCK = Mutex.new
```

Comment:
constant or class variable?

Reply:
I think either would work, but I generally try to avoid class variables. The `@@` syntax just feels so wrong